experimental/packages/opentelemetry-instrumentation-http/README.md

```diff
@@ -140,12 +139,12 @@ Attributes collected:
 | `http.url`                                  | Full HTTP request URL in the form `scheme://host[:port]/path?query[#fragment]` |
 | `http.user_agent`                           | Value of the HTTP User-Agent header sent by the client                         |
 | `net.host.ip`                               | Like net.peer.ip but for the host IP. Useful in case of a multi-IP host        |
-| `net.host.name`                             | Local hostname or similar                                                      |
-| `net.host.port`                             | Like net.peer.port but for the host port                                       |
-| `net.peer.ip.`                              | Remote address of the peer (dotted decimal for IPv4 or RFC5952 for IPv6)       |
-| `net.peer.name`                             | Remote hostname or similar                                                     |
-| `net.peer.port`                             | Remote port number                                                             |
-| `net.transport`                             | Transport protocol used                                                        |
+| `net.host.name`                             | Local hostname of the server.                                                  |
+| `net.peer.ip`                               | The IP address of the remote client                                            |
+| `net.peer.name`                             | The DNS name or IP address of the remote client                                |
+| `net.peer.port`                             | The port of the remote peer                                                   |
+| `peer.service`                              | The name of the remote service                                                |
+| `service.name`                              | Name of the service that generated the span                                     |
```

Comment:
I don't think we set service name in this instrumentation - that's a resource attribute.